builder/dockerfile/parser/testfiles/health/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM debian
+ADD check.sh main.sh /app/
+CMD /app/main.sh
+HEALTHCHECK
+HEALTHCHECK --interval=5s --timeout=3s --retries=1 --grace=4s --exit-on-unhealthy \
```

Comment:
Generally speaking retries should be multiple like 3 to avoid system instability. We don't want to provide examples of retries=1.